tests/introspection/iOS/iOSApiCtorInitTest.cs

```diff
@@ -180,6 +180,10 @@ protected override bool Skip (Type type)
 			// iOS 10 - this works only on devices, so we skip the simulator
 			case "MTLHeapDescriptor":
 				return Runtime.Arch == Arch.SIMULATOR;
+
+			case "NEPacketTunnelProvider":
+				return true;
+
```

Comment:
This is fine but in the wrong place. The bots reports the same error for XM, so it's the base class that needs to handle this (so both XI and XM ignore that .ctor).